gen-certs.sh

```diff
@@ -9,11 +9,16 @@ CITY=${CITY:-Nuremberg}
 STATE=${STATE:-Bavaria}
 COUNTRY=${COUNTRY:-DE}
 
-DIR="/etc/pki"
+DIR="${DIR:-/etc/pki}"
 CERTS="$DIR/_certs"
 PRIVATEDIR="$DIR/private"
 WORK="$DIR/_work"
 
+
+random_serial() {
+    python -c 'import random ; print(random.randint(1, 2**159))'
```

Comment:
We can easily avoid python for this.. something like `$(((RANDOM%10000000)+1))` - with a suitably high number in place of the `10000000`.

Reply:
It seems that `"The range of returned values is 0 to 32767"` for `$RANDOM`. Taking into account that the serial can be up to `2^160`, maybe it is not a big deal but it seems slightly short... 

However, I've found [this](https://unix.stackexchange.com/questions/157250/how-to-efficiently-generate-large-uniformly-distributed-random-integers-in-bas) with some other ways to generate random numbers. I'll find some other way.